css/details.css

```diff
@@ -12,13 +24,143 @@ th {
 }
 
 td {
-	text-align: right;
+	text-align: left;
+	padding-left: 1em;
+	max-width: 4em;
```

Comment:
This seems to be additional complexity brought in by specifying a `width` for the `<table>`. What about simply removing the `width` and adding a `padding` to the `th` and `td`? Or, if you want to fit the whole width of the `<details>`, set `width` to `100%`, add a `padding` and use the [`1%` trick](http://jsfiddle.net/MetalFrog/wexdX/) on the `td`?